{
    "config": {
        "error": {
            "cannot_connect": "Falha na liga\u00e7\u00e3o",
            "unknown": "Erro inesperado"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "Chave da API"
                }
            },
            "user": {
                "data": {
                    "url": ""
                }
            }
        }
    }
}